### DISCUSSION TITLE
plotItem.setLabel (units = 'what are all the possible values and definitions for units')

### DISCUSSION THREAD

**Discussion by pyKeith** [pyqtgraph/pyqtgraph#2767] [Q&A] [answered]
hi, I have been trying to get a reference list of all the acceptable values for units such as example>> units ='s' 

there are a few examples of some letters used however without explanation or definition.... or maybe my search capabilities are failing me!!

so any link to something well documented ? like this....

s = seconds  etc etc

or maybe I am not understanding how this parameter works ?

appreciate the help.
Cheers

**Comment by pijyoi** [comment, marked as answer]
Maybe this example might explain things:
```python
import pyqtgraph as pg
import numpy as np

pg.mkQApp()
pw = pg.PlotWidget()
pw.setLabel(axis='left', units='Eggs')
pw.plot(np.arange(0, 1_000_000_000, 1000))
pw.show()
pg.exec()
```

**Reply by pyKeith** [reply]
thank you........... clear !!  appreciate it !! sometimes you get tunnel vision and just need that little light !!   cheers !!!
